rpm/sensorfw-qt5-hybris.spec

```diff
@@ -37,18 +35,6 @@ Requires:   qt5-qtnetwork-devel
 %description devel
 Development headers for sensor framework daemon and libraries.
 
-
-%package configs
```

Comment:
After removing this package we need to do obsolete for this so that it gets removed from the images. Obsolestes: %{name}-configs <= 0.8.17 or so